app-containers/incus/incus-6.0.3-r2.ebuild

```diff
@@ -158,7 +158,8 @@ src_test() {
 src_install() {
 	export GOPATH="${S}/_dist"
 
-	if tc-is-cross-compiler ; then
+	export GOHOSTARCH=$(go-env_goarch "${CBUILD}")
+	if [ "${GOARCH}" != "${GOHOSTARCH}" ]; then
```

Comment:
We prefer double brackets:
https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101